src/core/core/model/story.py

```
@@ -1520,3 +1520,219 @@ def is_assigned(cls, story_id: str) -> bool:
+    id: Mapped[str] = db.Column(db.String(UUID_STR_LENGTH), primary_key=True, default=BaseModel.uuid7_str)
+    name: Mapped[str] = db.Column(db.String(120), nullable=False)
+    created: Mapped[datetime] = db.Column(db.DateTime, default=BaseModel.utcnow, nullable=False)
+    updated: Mapped[datetime] = db.Column(db.DateTime, default=BaseModel.utcnow, nullable=False)
+
+    user_id: Mapped[str] = db.Column(db.String(UUID_STR_LENGTH), db.ForeignKey("user.id", ondelete="CASCADE"), nullable=False, index=True)
+    user: Mapped["User"] = relationship("User")
+    stories: Mapped[list["Story"]] = relationship(
+        "Story", secondary="story_bookmark_story", cascade="save-update, merge", passive_deletes=True, single_parent=False, lazy="selectin"
+    )
+
+    def __init__(self, name: str, user_id: str, id: str | None = None, stories: list[str] | None = None):
```

Ruff (0.15.17)

[error] Function argument `id` is shadowing a Python builtin

(A002)